docs/Advanced-Usage/FAQs.rst

```diff
@@ -53,3 +53,25 @@ interactive simulation, or you've connected the NIC to the internet), this is
 likely a bug in one of your custom bridge implementations or in FireSim. In
 fact, for a given target-design, enabling printf synthesis, assertion synthesis,
 autocounter, or Auto ILA, should not change the simulated behavior of the machine.
+
+Is there a way to compress workload results when copying back to the manager instance?
+--------------------------------------------------------------------------------------
+
+FireSim doesn't support compressing workload results before copying them back to the manager instance.
+Instead we recommend that you use a modern filesystem (like ZFS) to provide compression for you.
+For example, if you want to use ZFS to transparently compress data:
+
+#. Attach a new volume to your EC2 instance (either at runtime or during launch).
+   This is where data will be stored in a compressed format.
+#. Make sure that the volume is attached (using something like ``lsblk -f``).
+   This new volume should not have a filesystem type and should be unmounted (named something like ``nvme1n1``).
+#. Install ZFS according to https://zfsonlinux.org/
```

Comment:
This should link directly to the CentOS installation procedures.
```suggestion
#. Install ZFS according to <https://openzfs.github.io/openzfs-docs/Getting%20Started/RHEL-based%20distro/index.html>.
   Check `/etc/redhat-release` to verify the CentOS version of the manager instance.
```

The current AMI is based on CentOS 7.9.  The instructions I used were:
```sh
yum install https://zfsonlinux.org/epel/zfs-release.el7_9.noarch.rpm
rpm --import /etc/pki/rpm-gpg/RPM-GPG-KEY-zfsonlinux
yum install epel-release kernel-devel zfs
```